src/components/Navbar/components/NavbarRightList/NavbarLoggedInMenu/NavbarLoggedInMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import Button from "@/components/Button/Button"
+import { HandleMenuClickProps } from "../NavbarRightList"
+import { theme } from "@/styles/theme"
+
+interface NavbarNotLoggedInMenuProps {
+  handleMenuClick: HandleMenuClickProps
+}
+const NavbarLoggedInMenu = ({
+  handleMenuClick,
+}: NavbarNotLoggedInMenuProps) => {
```

Comment:
타입 재사용은 이렇게 하면 되는군요?? 훈오님 코드 보면서 많이 배웁니다! 👍
하나 제안드리고 싶은게 있는데 NavbarLoggedInMenu인데 interface는 NavbarNotLoggedInMenuProps라서 혹시 놓치신 것 같아서 말씀드려봅니다! 😊
```suggestion
interface NavbarLoggedInMenuProps {
  handleMenuClick: HandleMenuClickProps
}
const NavbarLoggedInMenu = ({
  handleMenuClick,
}: NavbarLoggedInMenuProps) => {
```

Reply:
> 타입 재사용은 이렇게 하면 되는군요?? 훈오님 코드 보면서 많이 배웁니다! 👍 하나 제안드리고 싶은게 있는데 NavbarLoggedInMenu인데 interface는 NavbarNotLoggedInMenuProps라서 혹시 놓치신 것 같아서 말씀드려봅니다! 😊

그러게요.. 잘못됐네요!! 감사합니다 ㅎㅎ